### PR TITLE
Build runner

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Matches previous configuration in pubspec.yaml - transformers - $dart2js
+targets:
+  $default:
+    builders:
+      build_web_compilers|entrypoint:
+        # These are globs for the entrypoints you want to compile.
+        generate_for:
+        - web/**.dart
+        options:
+          compiler: dart2js
```
Comment (jakemac53) [inline build.yaml]: no ddc config?

Comment (matanlurey) [reply build.yaml]: Definitely should standardize on doing dart2js as `--config=release`.

(Or if we add `pub run build_server deploy`, or something)

File: README.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ in [Dart](https://www.dartlang.org).
...
+ * Sound Effects - Alistair Hirst, [OMNI Audio](https://www.linkedin.com/in/alistairhirst/)
  * Winning Sax Riff - Brian Moore
+
+## Running and building
+
+ * With a [Dart 2 SDK](https://www.dartlang.org/dart-2) installed.
+ * Clone the repository.
+ * Run `pub get`.
+ * To run locally: `pub run build_runner serve`
+ * To build: `pub run build_runner build --output build`
```
Comment (jakemac53) [inline README.md]: `--config release` now?

**Comment by kevmoo** [comment]
@jakemac53 – PTAL

File: build.release.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# See https://github.com/dart-lang/build/tree/master/build_web_compilers#configuration
+# Matches previous configuration in pubspec.yaml - transformers - $dart2js
```
Comment (natebosch) [inline build.release.yaml]: [nit] Something to consider in the future - comments about "previous" stuff are pretty temporal, their importance drops over time but they stick in the code forever. Consider keeping comments like this in the commit message instead.

File: build.release.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  $default:
+    builders:
+      build_web_compilers|entrypoint:
+        # These are globs for the entrypoints you want to compile.
+        generate_for:
+        - web/**.dart
+        options:
+          compiler: dart2js
+          dart2js_args:
+            # Align with the default configuration of dart2js in `pub build`
```
Comment (natebosch) [inline build.release.yaml]: does this align with `pub build` defaults? I wouldn't think some of these would be the default...

File: build.release.yaml (hunk 3 above)
Comment (kevmoo) [reply build.release.yaml]: Ack.

File: build.release.yaml (hunk 4 above)
Comment (kevmoo) [reply build.release.yaml]: Ooops... Yeah. Now they don't. Will fix...

**Pull request merged**